3.11/aql/invocation-with-arangosh.md

```diff
@@ -350,6 +390,25 @@ queries. Additionally, query statistics, warnings, and profiling data is only
 available after the query has finished. The default value is `false`.
 {% endhint %}
 
+### `retriable`
+
+Specify `true` as a field of `options` and the request to retrieve the result from
```

Comment:
Looks like an 95% copy of the text in https://github.com/arangodb/docs/pull/1234/files#diff-779df756d3121cc95e344ecf5ae56eeaa531830eb739975dee96332155c34280R340.
The same comments apply here.